### PR TITLE
Minor bug

### PR DESCRIPTION
**Pull request by laullon**

### CODE REVIEW

File: wavefront_cli/lib/proxy.py [hunk 1]
```diff
@@ -114,7 +114,7 @@ def configure_proxy(url, wavefront_api_token, csp_api_token=None,
     if csp_api_token:
         # replace csp api token
         cmd = ('sed', '-i', '-e',
-               f'/#cspAPIToken=/c\tcspAPIToken={csp_api_token})',
+               f'/#cspAPIToken=/c\tcspAPIToken={csp_api_token}',
```
Comment (ustinov) [inline wavefront_cli/lib/proxy.py:117]:
```suggestion
               f'/#cspAPIToken=/c\tcspAPIToken={csp_api_token} ',
```

Comment (ustinov) [reply wavefront_cli/lib/proxy.py:117]: We'd also need a space

Comment (laullon) [reply wavefront_cli/lib/proxy.py:117]: for what ??

Comment (ustinov) [reply wavefront_cli/lib/proxy.py:117]: Thought it was followed by a continuation string, but I see there's a comma, so we are fine

**Review by ustinov — APPROVED** [review]

**Pull request merged**